plugin/evm/vm.go

```diff
@@ -977,6 +1020,7 @@ func (vm *VM) Shutdown(context.Context) error {
 	if vm.ctx == nil {
 		return nil
 	}
+	vm.closeCtxCancel()
```

Comment:
I can probably use something like a shtudown channel + waitgroup like we do for other background goroutines for consistency instead of this context pattern.